src/user/configure-programs-in-the-maintenance-app.md

```diff
@@ -372,8 +372,9 @@ program. A program needs several types of metadata that you create in the **Main
 | **Allow future incident dates** |Select checkbox if you want to allow the incident date in the program to be on a future date.|
 | **Only enroll once (per tracked entity instance lifetime)** |Select checkbox if you want a tracked entity to be able to enroll only once in a program. This setting is useful for example in child vaccination or post-mortem examination programs where it wouldn’t make sense to enroll a tracked entity more than once.|
 | **Show incident date** |This setting allows you to show or hide the incident date field when a tracked entity enroll in the program.|
-| **Description of incident date** | Type a description of the incident date<br> <br>For example:<br> <br>In an immunization program for child under 1 year old, the incident date is the child's birthday. <br> <br>In a maternal program, the incident date is the date of last menstrual period.  |
-| **Type a description of the enrollment date** |The date when the tracked entity is enrolled into the program|
+| **Custom label for incident date** | Type a description of the incident date<br> <br>For example:<br> <br>In an immunization program for child under 1 year old, the incident date is the child's birthday. <br> <br>In a maternal program, the incident date is the date of last menstrual period.  |
+| **Custom label for enrollment date** |The date when the tracked entity is enrolled into the program|
+| **Custom label for:**<br> - **enrollment** <br> - **event** <br> - **program stage** <br> - **follow-up** <br> - **registering unit** <br> - **relationship** <br> - **note** <br> - **tracked entity attribute** | These custom labels will, on a program-specific level, replace these terms in certain DHIS2 apps. It is important to note that this configuration does not distinguish between singular and plural, so the label should consider this. These labels are currently only consumed in the DHIS2 Android app. |
```

Comment:
> These labels are currently only consumed in the DHIS2 Android app. 
I think it's clearer to reword this to put the focus on the "currently", for example:
"Currently, custom labels are only used by the DHIS2 Android app."